bundles/core/src/test/java/com/adobe/cq/wcm/core/components/commons/link/LinkManagerTest.java

```diff
@@ -73,7 +73,7 @@ void testResourceExternalLink() {
         context.currentResource(linkResource);
         Link link = getUnderTest().get(linkResource).build();
 
-        assertValidLink(link, "http://myhost");
+        assertValidLink(link, "http://myhost",context);
```

Comment:
Spacing is off here (and in other places too).

Reply:
good point, fixed it.